dependency_repos.repos

```diff
@@ -1,9 +1,9 @@
 repositories:
   mocap_msgs:
     type: git
-    url: https://github.com/MOCAP4ROS2-Project/mocap_msgs.git
```

Comment:
Please, remove this file from your commit. I suppose it is related to the fork you did.

Reply:
Done.